src/Compiler/OnnxMlirCompiler.cpp

```diff
@@ -0,0 +1,45 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "OnnxMlirCompiler.h"
+#include "CompilerUtils.hpp"
+
+void SetCompileContext(
```

Comment:
You can check elsewhere, but we are also trying to use small cap for the function, and we like them to be verbs, so `setCompileContext` would be in the spirit of things